features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/CheckableCompoundButtonMapper.kt

```diff
@@ -36,27 +40,95 @@ internal abstract class CheckableCompoundButtonMapper<T : CompoundButton>(
     @UiThread
     override fun resolveCheckableBounds(view: T, pixelsDensity: Float): GlobalBounds {
         val viewGlobalBounds = viewBoundsResolver.resolveViewGlobalBounds(view, pixelsDensity)
-        var checkBoxHeight = DEFAULT_CHECKABLE_HEIGHT_IN_PX
+        var checkBoxHeight = DEFAULT_CHECKABLE_HEIGHT_IN_DP * pixelsDensity.toLong()
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
             view.buttonDrawable?.let {
                 checkBoxHeight = it.intrinsicHeight.toLong()
             }
         }
-        // minus the padding
-        checkBoxHeight -= MIN_PADDING_IN_PX * 2
         checkBoxHeight = checkBoxHeight.densityNormalized(pixelsDensity)
         return GlobalBounds(
-            x = viewGlobalBounds.x + MIN_PADDING_IN_PX.densityNormalized(pixelsDensity),
```

Comment:
why we don't account for padding anymore?

Reply:
Before we drew our own shape to display the square of the checkbox, so the padding should be used. but now we directly use the drawable inside `CompoundButton`, which already has a inset padding, we don't need to add an additional one.
![image](https://github.com/DataDog/dd-sdk-android/assets/15606345/3f79b092-1990-474f-aa9b-f5a5a9382ee5)